pytential/qbx/direct.py

```diff
@@ -78,8 +78,7 @@ def get_kernel(self):
                 <> icenter = qbx_center_numbers[itgt]
                 <> itgt_overall = qbx_tgt_numbers[itgt]
 
-                <> a[idim] = center[idim, icenter] - src[idim, isrc] \
-                        {dup=idim}
+                <> a[idim] = center[idim, icenter] - src[idim, isrc]
                 <> b[idim] = tgt[idim, itgt_overall] - center[idim, icenter] \
                         {dup=idim}
```

Comment:
This was duplicating all the uses of `idim`, which then was unused and [this check in loopy](https://github.com/inducer/loopy/blob/f0af37c3d94393e02f4fd648d907b49a85f56b27/loopy/check.py#L243) warned about it.

Reply:
LGTM. Thanks for pointing it out!